frontend/src/helpers/Api/types.ts

```diff
@@ -8,6 +8,7 @@ export interface IApi {
     next?: any
     params?: Record<string, any>
     removeCacheTimeout?: number
+    skip?: boolean
```

Comment:
The key below, `triggerIf`, does exactly what we need, no need for `skip`, you can delete it.